cmd/csi/server/builder.go

```diff
@@ -5,7 +5,6 @@ import (
 	cmdManager "github.com/Dynatrace/dynatrace-operator/cmd/manager"
```

Comment:
Out of curiosity:

I deployed the operator based on this branch, and had a look into the csi server. I saw that there are still .db files. I guess they will be removed in future steps, right? 

```
bash-5.1# pwd
/data
bash-5.1# ls
asj34817  codemodules  csi.db  csi.db-shm  csi.db-wal  dynakube
```

Reply:
just leftovers from pervious install nothing more, we will either clean it up later, or just leave it as is